chain/cosmos/chain_node.go

```diff
@@ -1066,13 +1072,10 @@ func (tn *ChainNode) ExportState(ctx context.Context, height int64) (string, err
 	var (
 		doc     = "state_export.json"
 		docPath = path.Join(tn.HomeDir(), doc)
-
-		isNewGenesis = tn.Chain.Config().UsingNewGenesisCommand
-
 		command = []string{"export", "--height", fmt.Sprint(height), "--home", tn.HomeDir()}
 	)
 
-	if isNewGenesis {
```

Comment:
we can keep the var here so the lookup doesn't need to be performed twice within this method

Reply:
27417615232c08a5ff32f5a83bee5ea6259383cd